src/rosdiscover/config.py

```diff
@@ -27,15 +81,18 @@ class Config:
         A set of environment variables that should be used by the application.
     app: roswire.app.App
         The ROSWire application for this configuration.
+    node_sources: List[NodeSourceInfo]
+        The list of information about the sources for nodes
```

Comment:
Maybe rewrite to: "Provides instructions on how to statically recover the architecture code for individual nodes."

Reply:
DOne